lib/add_bc_user.sh

```diff
@@ -1,15 +1,29 @@
 #!/bin/bash -e
 
 echo "This script add's a user with password to the bridghead"
+
+if [ $# -eq 0 ]; then
+    echo "No arguments provided, please provide the prject name"
+    exit 1
+fi
+
+if [ ! -f /etc/systemd/system/bridgehead@$1.service.d/override.conf ]; then
+    echo "Please create a Service first, with setup-bridgehead-units.sh"
+    exit
+fi
+
 read -p 'Username: ' bc_user
 read -sp 'Password: ' bc_password
 
-echo 
+echo
+
+
+bc=`docker run --rm -it httpd:latest htpasswd -nb $bc_user $bc_password`
```

Comment:
Short question about the naming: what is the meaning behind "bc"?

Reply:
basic_user